### PR TITLE
Issue160-DemConfigAPI

### PR DESCRIPTION
**Pull request by susanodom**
pull request for issue 160

### CODE REVIEW

**Comment by aguibert** [comment]
@Emily-Jiang did you have any thoughts on this PR?  It looks like a good initial demonstration of MP Config in the Vote service.

**Comment by heiko-braun** [comment]
For those who don't run on cloud foundry this seems a bit too specific (VCAP_*). I like the idea of using mp-config with the services, but would prefer a vendor or target environment neutral setup.

**Comment by Emily-Jiang** [comment]
@aguibert the changes are fine but the pom dependency should be against mpConfig-1.1.
@heiko-braun Since vote service was written to be deployed to cloud foundry, it makes sense to use vcap. By the way, if VCAP does not exist, it will fall back to cloudant connection. This PR did not introduce VCAP but just try to inject the config.

File: microservice-vote/pom.xml [hunk 1]
```diff
@@ -193,6 +193,12 @@
             <artifactId>demo-bootstrap</artifactId>
             <version>${project.version}</version>
         </dependency>
+        <dependency>
+            <groupId>net.wasdev.wlp.starters.ms-builder</groupId>
```
Comment (Emily-Jiang) [inline microservice-vote/pom.xml]: Please use the following snippet 
<dependency>
    <groupId>org.eclipse.microprofile.config</groupId>
    <artifactId>microprofile-config-api</artifactId>
    <version>1.1</version>
</dependency>

Comment (Emily-Jiang) [inline microservice-vote/pom.xml]: should be

```
<dependency>
    <groupId>org.eclipse.microprofile.config</groupId>
    <artifactId>microprofile-config-api</artifactId>
    <version>1.1</version>
</dependency>
```

File: microservice-vote/pom.xml [hunk 2]
```diff
@@ -331,6 +338,22 @@
...
+               		<execution>
+           				<id>install-feature</id>
+           				<phase>prepare-package</phase>
+          				<goals>
+               				<goal>install-feature</goal>
+           				</goals>
+          				<configuration>
+             		  	  	<features>
+               		    		<acceptLicense>true</acceptLicense>
+                  	 			<feature>mpConfig-1.0</feature>
```
Comment (Emily-Jiang) [inline microservice-vote/pom.xml]: update to
 `
<feature>mpConfig-1.1</feature>
`

Comment (aguibert) [reply microservice-vote/pom.xml]: @Emily-Jiang Liberty 17.9.0.0 only provides mpConfig-1.0, so we cannot update to mpConfig-1.1 yet.

**Comment by aguibert** [comment]
This PR can be closed, as it has already been merged under PR #172

**Comment by Emily-Jiang** [comment]
close as per the comments from @aguibert

**Pull request closed without merging**